android/CMakeLists.txt

```diff
@@ -23,9 +27,8 @@ else()
   set(GOARCH "please-fail :/")
 endif()
 
-set(WORMHOLE_WILLIAM_GO_DIR ${CMAKE_SOURCE_DIR}/../wormhole-william)
-
-set(CC_FOR_TARGET ${ANDROID_TOOLCHAIN_ROOT}/bin/${CMAKE_C_COMPILER_TARGET}-clang)
+set(CC_FOR_TARGET
+    ${ANDROID_TOOLCHAIN_ROOT}/bin/${CMAKE_C_COMPILER_TARGET}-clang)
 
 # Fix the clang triplet used as parameter for -target flag. compiler with -none-
 # are not in the ndk
```

Comment:
Figure out if we could clean these text replacements on `CC_FOR_TARGET`

Reply:
Tried removing either one and the issue remains; these replacements seem needed for now